### PR TITLE
chore: Add tauri-plugin-websocket history

### PR DESCRIPTION
**Pull request by JonasKruckenberg**
This PR adds the history of the `websocket` plugin for better git blame and for preserving history like #72

### CODE REVIEW

File: plugins/single-instance/examples/vanilla/src-tauri/Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,7 @@
+[workspace]
+
 [package]
-name = "single-instance-example"
+name = "app"
```
Comment (Beanow) [inline plugins/single-instance/examples/vanilla/src-tauri/Cargo.toml]: Looks like a change in different plugin.

File: plugins/sql/src/plugin.rs [hunk 2]
```diff
@@ -348,7 +348,9 @@ impl Builder {
                     drop(lock);
 
                     app.manage(instances);
-                    app.manage(Migrations(Mutex::new(self.migrations.take().unwrap_or_default())));
+                    app.manage(Migrations(Mutex::new(
+                        self.migrations.take().unwrap_or_default(),
+                    )));
```
Comment (Beanow) [inline plugins/sql/src/plugin.rs:353]: Ditto, change in other plugin.

Comment (JonasKruckenberg) [reply plugins/sql/src/plugin.rs:353]: it's `cargo fmts` fault 🤷🏻‍♂️ 😂

File: plugins/single-instance/examples/vanilla/src-tauri/Cargo.toml (hunk 1 above)
Comment (JonasKruckenberg) [reply plugins/single-instance/examples/vanilla/src-tauri/Cargo.toml]: true, though the name really doesn't matter much

Comment (JonasKruckenberg) [reply plugins/single-instance/examples/vanilla/src-tauri/Cargo.toml]:
```suggestion
name = "single-instance-example"
```

File: plugins/sql/src/plugin.rs (hunk 2 above)
Comment (FabianLars) [reply plugins/sql/src/plugin.rs:353]: And that's why i should stop using the web editor for small changes in rust files :))

**Pull request merged**